headers/util.ts

```diff
@@ -57,21 +55,17 @@ export function serializeCsp(x: Csp): string {
 }
 
 /**
- * Recursively collects all the CSPs exported by scripts in a certain directory.
- * @param dir Path to the directory to search.
+ * Collects all the CSPs exported by the csps/index.ts file.
+ * @param dir Path to the directory to search (no longer used but kept for backward compatibility).
  */
-export function collectCsps(dir: string): Csp[] {
-  const out = []
-  for (const item of fs.readdirSync(path.resolve(__dirname, dir), { withFileTypes: true })) {
-    const itemPath = `${dir}/${item.name}`
-    if (item.isDirectory()) {
-      out.push(...collectCsps(itemPath))
-    } else {
-      if (!item.isFile() || !/\.(j|t)s$/.test(item.name)) continue
-      const csp = require(itemPath).csp
-      if (!csp) throw new Error(`expected ${itemPath} to export a member named 'csp'`)
-      out.push(csp)
+export function collectCsps(_dir: string): Csp[] {
```

Comment:
seems like a pointless helper function now, just import `csps` directly in `headers/index.ts`?

Reply:
Removed in [780fc35](https://github.com/shapeshift/web/pull/8924/commits/780fc358dd698fa1d4f35579acc5f620dc5da98a)